Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/DiagnosticInformationCollectingRequestErrorDetectionStrategy.cs

```diff
@@ -18,20 +18,20 @@
 using Coypu;
 using Remotion.Web.Development.WebTesting.RequestErrorDetectionStrategies;
 
-namespace Remotion.Web.Development.WebTesting.IntegrationTests
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure
 {
   /// <summary>
   /// Testclass to be able to seperate Tests for <see cref="IRequestErrorDetectionStrategy"/> implementation and calls.
   /// </summary>
-  public class DiagnosticInformationCollectioningRequestErrorDetectionStrategy : IRequestErrorDetectionStrategy
+  public class DiagnosticInformationCollectingRequestErrorDetectionStrategy : IRequestErrorDetectionStrategy
```

Comment:
to make the fture reading of the Feature Branch easer: we should move the changes from the rename to a separate commit. Also, do we still need this in the Infrastructure project now?